akka-ssl-config/src/main/scala/com/typesafe/sslconfig/akka/util/AkkaLoggerBridge.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package com.typesafe.sslconfig.akka.util
```

Comment:
yeah let's move it to `com.lightbend`, to avoid weird classpath issues I think, as Patrik suggested